README.md

```diff
@@ -104,16 +109,35 @@ Options
    --no-cache                  disable cache
 ```
 
-To find possible transform places:
+To find possible transform places in a folder named "src", run:
+
+```
+npx putout src
+```
+
+To find possible transform places in multiple folders, such as folders named "lib" and "src", run: 
 
 ```
-npx putout lib test
+npx putout lib src
```

Comment:
Would be great to keep `test` here, because most packages in this `monorepo` has this folder, but none of them has folder `src`

Reply:
Switched back to `lib` and `test`